docroot/modules/custom/va_gov_workflow/src/EventSubscriber/EntityEventSubscriber.php

```diff
@@ -265,26 +264,31 @@ public function entityDelete(EntityDeleteEvent $event): void {
    * @param string $form_id
    *   The form id.
    */
-  public function requireRevisionMessage(array &$form, FormStateInterface &$form_state, $form_id) {
-    // Check if the block-content form is of type promo and exit (#15187)
+  public function requireRevisionMessage(array &$form, FormStateInterface &$form_state, $form_id): void {
+    // Hide the checkbox that lets user opt into making a revision.
     $formObject = $form_state->getFormObject();
     if ($formObject instanceof EntityFormInterface) {
       $entity = $formObject->getEntity();
-      if ($entity instanceof BlockContentInterface) {
-        if ($entity->bundle() === 'promo') {
-          return;
-        }
+      if ($entity->bundle() === 'promo') {
```

Comment:
Minor:  You had good instincts before to make this check specific to blocks named promo.   We don't have a node named promo or a menu named promo but it is a simple machine name that could easily appear so it makes sense to keep your original check.
```suggestion
      if (($entity instanceof BlockContentInterface) && ($entity->bundle() === 'promo')) {
```